sdk/storage/Azure.Storage.Files.Shares/api/Azure.Storage.Files.Shares.netstandard2.0.cs

```diff
@@ -611,6 +624,18 @@ public partial class ShareFileCopyInfo
         public Azure.ETag ETag { get { throw null; } }
         public System.DateTimeOffset LastModified { get { throw null; } }
     }
+    public partial class ShareFileCopyOptions
+    {
+        public ShareFileCopyOptions() { }
+        public Azure.Storage.Files.Shares.Models.ShareFileRequestConditions Conditions { get { throw null; } set { } }
+        public Azure.Storage.Files.Shares.Models.CopyableFileSmbProperties CopyableFileSmbProperties { get { throw null; } set { } }
```

Comment:
I think `CopyableFileSmbProperties` works well for the type name, but I don't know if it's as clear for the property name.  What about something like `public CopyableFileSmbProperties SmbPropertiesToCopy { get; set; }` so then this new property aligns in intellisense?

Reply:
Fixed.